runtime/compiler/control/rossa.cpp

```diff
@@ -1654,6 +1657,24 @@ onLoadInternal(
       if (!JITServer::loadLibsslAndFindSymbols())
          return -1;
       }
+#if defined(PERSISTENT_LOGGING_SUPPORT)
+//TODO: FIx this flag
+   //if(TR::Options::getCmdLineOptions()->getOption(TR_PersistLogging))
```

Comment:
This option should be used at the client in my opinion. The server should be controlled by those `-XX:` options. Do we need another option?